tests/entry_page_test.js

```diff
@@ -2,6 +2,45 @@ const utils = require('../utils/utils');
 
 Feature('entry_page');
 
+const logExperimentDetails = (
+    countOfActiveKeys, countOfEntryPoints, allActiveKeysEntryPointInfo, countOfConfirmations, confirmations
+) => {
+    console.log(`Active keys: ${countOfActiveKeys}`);
+    console.log(`Entry points: ${countOfEntryPoints}`);
+    allActiveKeysEntryPointInfo.forEach(key => {
+        console.log(`${key.id} | isEntryPoint: ${key.isEntryPoint}`);
+    })
+    console.log(`Confirmations: ${countOfConfirmations}`);
+    console.log(confirmations);
+}
+
+const validateEntryPoints = (allActiveKeysEntryPointInfo, entryPointValue) => {
+    let isValid;
+    allActiveKeysEntryPointInfo.forEach(key => {
+        if (key.isEntryPoint === entryPointValue) {
+            isValid === false ? isValid = false : isValid = true;
+        } else {
+            isValid = false;
```

Comment:
You can return early here to avoid the semi- complex logic needed to keep isValid false
Something like
if (key.isEntryPoint !== entryPointValue) {
return false
}
....